### ISSUE TITLE
Feedback from Phil

### ISSUE THREAD

**Issue by PhilChodrow** [evanflaks/451Project#1]
Thanks team, looks pretty interesting! For this project, since it's primarily using a pretty clean data set from Kaggle, I'd like you to focus in on methods some. There's a cool opportunity to do this because you are aiming to predict not just one but **two** numbers for a given observation. This leads to the idea of *multi-objective regression,* in which your model produces two predictions which should then be compared to the two targets. One way to approach this would be to design your own machine learning model with a customized loss function (which accepts two scores and two targets) and then train it. Use of neural networks (which we'll start covering soon) would be one natural way to approach this, but since you both have math background you could also build a setup from scratch by implementing a score function with two outputs, a loss function for comparison, and a corresponding gradient. 

There may be other ways to add depth to the project -- talk to me if you want to develop a different idea (or if you just want to chat more about this one). 

Interesting, team! Looking forward to seeing what you come up with here.
